build.gradle

```diff
@@ -89,36 +195,23 @@ bintray {
         name = 'fees-client'
         userOrg = 'hmcts'
         licenses = ['MIT']
-        vcsUrl = gitRepo
+        vcsUrl = 'https://github.com/hmcts/fees-java-client'
         version {
             name = project.version
         }
     }
 }
 
-def versions = [
-        logback           : '1.2.3',
-        springBoot        : springBootVersion,
-        springfoxSwagger  : '2.9.2',
-        reformJavaLogging : '5.0.1'
-]
-
-dependencies {
-    compile group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
-    compile group: 'io.springfox', name: 'springfox-swagger-ui', version: versions.springfoxSwagger
-
-    compile group: 'ch.qos.logback', name: 'logback-classic', version: versions.logback
-    compile group: 'ch.qos.logback', name: 'logback-core', version: versions.logback
-    compile group: 'uk.gov.hmcts.reform', name: 'java-logging', version: versions.reformJavaLogging
-
-    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: '5.5.1'
-    testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: '5.5.1'
-}
-
 bootJar {
     enabled = false
 }
 
 jar {
     enabled = true
+    manifest {
+        attributes(
+                'Class-Path': configurations.compile.collect { it.getName() }.join(' '),
+                'Main-Class': 'Main'
```

Comment:
I don't think we need to make this JAR an `executable` jar, as this is a library. We can add  `example` or `sample usage` in a separate module in the same repo.